ui/src/main/webapp/src/app/shared/validators/project-name-not-exists.validator.ts

```diff
@@ -29,6 +29,7 @@ export class ProjectNameNotExistsValidator implements Validator {
                 if (result == null) {
                     resolve(null);
                 } else {
+                    // When updating, it's ok to enter the same name.
                     if (this.project != null && result == this.project.id)
                         resolve(null);
                     else
```

Comment:
A comment here is definitely unrelated to the about page. I think that is perfectly fine for a comment though. Useful comments are good regardless of whether they are in a relevant PR. :)